src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts

```diff
@@ -299,6 +304,16 @@ export class AcrossConfigStoreClient extends BaseAbstractClient {
         // class itself to work with the generalized ConfigStore.
         const l1Token = args.key;
 
+        // For now, we'll need to check if this value is undefined. In the
+        // future, we should make this a required field.
+        if (parsedValue.uba !== undefined) {
```

Comment:
Can we instead set default values? We can add default values to the UMIP so this is in-protocol

Reply:
Yep! These will be needed to be supplied by Data. I'll create a ticket for it